api/app/src/main/kotlin/packit/security/WebSecurityConfig.kt

```diff
@@ -26,8 +29,18 @@ import packit.security.oauth2.OAuth2UserService
 import packit.security.provider.JwtIssuer
 import packit.service.BasicUserDetailsService
 
+@ConditionalOnProperty(prefix = "auth", name = ["enabled"], havingValue = "true", matchIfMissing = true)
```

Comment:
Not a huge fan of having two different places that interpret the `auth.enabled` property, once here and once in `AppConfig`.

I believe something like this should work instead:
```kt
@ConditionalOnExpression("@appConfig.authEnabled")
```

This way the `AppConfig` remains the only source of truth for whether auth is enabled or not.